x-pack/plugins/aiops/public/cases/register_change_point_charts_attachment.tsx

```diff
@@ -19,7 +22,11 @@ export function registerChangePointChartsAttachment(
   coreStart: CoreStart,
   pluginStart: AiopsPluginStartDeps
 ) {
-  const EmbeddableComponent = getEmbeddableChangePointChart(coreStart, pluginStart);
+  const EmbeddableComponent = getEmbeddableChangePointChart(
```

Comment:
Do we still need to disable addition to a case if no charts are selected?

<img width="244" alt="image" src="https://github.com/elastic/kibana/assets/7405507/6b74f5dd-d272-4269-85cb-1482fc6653db">

Could we use the same workflow as for Dashboard where the user first selects whether to add charts or the table?

Reply:
Also can we hide the context menu when the 'Create case' flyout opens? I'm finding the menu stays visible till I navigate to the new case:

<img width="840" alt="image" src="https://github.com/elastic/kibana/assets/7405507/6e8977fb-ddad-444d-a367-8545f2635bd9">

Reply:
While attaching to a Case, we always preserve a time range, and I also assumed we need specific partitions related to the case, e.g. metric distribution changed for a particular host. 
We can change the workflow to be the same as adding to a dashboard. In that case, change points are computed for all partitions, and if data for some partitions is ingested later, displayed results in the change point attachment could change accordingly.

Reply:
> While attaching to a Case, we always preserve a time range, and I also assumed we need specific partitions related to the case

Sorry - you're correct, we should only enable the Case action when specific partitions are selected. But currently it always adds as charts - should we allow the user to add as a table view, especially when multiple results are selected?

Reply:
Added in https://github.com/elastic/kibana/pull/174348/commits/00367d4922d44540f37c63a2fb7649426cfcbf43